airflow/models/dagrun.py

```diff
@@ -113,8 +113,9 @@ def refresh_from_db(self, session=None):
     @staticmethod
     @provide_session
     def find(dag_id=None, run_id=None, execution_date=None,
+             execution_date_before=None, execution_date_after=None,
              state=None, external_trigger=None, no_backfills=False,
-             session=None):
+             session=None, state_not_equal=None):
```

Comment:
Make session the last param always please.

Reply:
done.